sklearn/ensemble/weight_boosting.py

```diff
@@ -863,9 +864,10 @@ class AdaBoostRegressor(BaseWeightBoosting, RegressorMixin):
 
     Parameters
     ----------
-    base_estimator : object, optional (default=DecisionTreeRegressor)
+    base_estimator : object, optional (default=None)
         The base estimator from which the boosted ensemble is built.
-        Support for sample weighting is required.
+        Support for sample weighting is required. If we chose the default
+        value, then the base estimator is DecisionTreeClassifier(max_depth=3).
```

Comment:
(1) ``If we chose the default value`` -> If ``None``
(2) DecisionTreeClassifier -> DecisionTreeRegressor
(3) Please use double backticks around  DecisionTreeRegressor(max_depth=3)